poetry/patterns/single.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Title: single
+ * Slug: poetry/single
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"padding":{"top":"96px","right":"0","bottom":"96px","left":"0"},"blockGap":"40px"}},"layout":{"type":"flex","flexWrap":"nowrap","orientation":"horizontal","verticalAlignment":"top","justifyContent":"center"}} -->
+    <div class="wp-block-group" style="padding-top:96px;padding-right:0;padding-bottom:96px;padding-left:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"96px"}}},"layout":{"type":"default"}} -->
+        <div class="wp-block-group" style="padding-top:96px"><!-- wp:post-date /--></div>
+        <!-- /wp:group -->
+
+        <!-- wp:group {"style":{"spacing":{"padding":{"right":"56px","left":"56px"},"blockGap":"32px"},"border":{"left":{"width":"2px"}}},"layout":{"type":"flex","orientation":"vertical","flexWrap":"nowrap"}} -->
+        <div class="wp-block-group" style="border-left-width:2px;padding-right:56px;padding-left:56px"><!-- wp:image {"id":15,"width":"56px","height":"56px","sizeSlug":"full","linkDestination":"none"} -->
+            <figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url(get_stylesheet_directory_uri()); ?>/assets/images/feather-small.png" alt="" class="wp-image-15" style="width:56px;height:56px" /></figure>
```

Comment:
Check the [guidelines for pattern creation](https://github.com/WordPress/community-themes?tab=readme-ov-file#pattern-creation-guidelines):

> To create dynamic image links in your block patterns, utilize the `get_template_directory_uri()` function. This function retrieves the URL of the current theme's directory, ensuring that the image links are relative to the theme and work correctly even if the website's directory structure changes or if we are using a child theme. This is essential for maintaining the stability and portability of your patterns.